project/src/main/java/com/google/sps/servlets/AuthServlet.java

```diff
@@ -22,17 +22,55 @@
 import com.google.gson.Gson;
 import com.google.sps.data.*;  
 
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.Connection;
+import java.sql.SQLException;
+import javax.sql.DataSource;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
 @WebServlet("/authentication")
 public class AuthServlet extends HttpServlet {
 
+  private static final Logger LOGGER = Logger.getLogger(AuthServlet.class.getName());
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    User userObj = User.create(1896,"Phoebe Khokgawe","phoebek@");
+    DataSource pool = (DataSource) request.getServletContext().getAttribute("my-pool");
 
-    Gson gson = new Gson();
-    
-    response.setContentType("application/json");
+    try (Connection conn = pool.getConnection()) {
+      String email = request.getParameter("email");
+      User user = getUser(conn, email);
+
+      Gson gson = new Gson();
+      response.setContentType("application/json");
+      response.getWriter().println(gson.toJson(user));
+      //response.setContentType("text/html;");
+      //response.getWriter().println("<h1>" + user + "</h1>");
+    } catch (SQLException ex) {
+      LOGGER.log(Level.WARNING, "Error while attempting to find user.", ex);
+      response.setStatus(500);
+      response.getWriter().write("Unable to successfully fetch user");
+    }
+  }
 
-    response.getWriter().println(gson.toJson(userObj));
+  private User getUser(Connection conn, String email) throws SQLException {
+    String stmt = "SELECT id, name FROM users WHERE email='" + email + "';";
+    try (PreparedStatement userStmt = conn.prepareStatement(stmt);) {
+      // Execute the statement
+      ResultSet userRs = userStmt.executeQuery();
```

Comment:
`useRs`->`users`?

Reply:
userRs, for user resultset. Can change to userRS?

Reply:
Modified to rs, seems to be convention.